app/authenticators/custom-jwt.js

```diff
@@ -0,0 +1,66 @@
+import JWT from 'ember-simple-auth-token/authenticators/jwt';
+import { get } from '@ember/object';
+import { assign } from '@ember/polyfills';
+import { Promise } from 'rsvp';
+import { isEmpty } from '@ember/utils';
+import ENV from 'open-event-frontend/config/environment';
+import { inject as service } from '@ember/service';
+
+export default JWT.extend({
+  init() {
+    this._super(...arguments);
+  },
+
+  session: service(),
+
+  handleAuthResponse(response, oldRefreshToken = null) {
```

Comment:
Why is this needed

Reply:
handleAuthresponse handle response and set refresh token.
It expect refresh token from every refresh request.

It take refresh token from first request.
and we use that token in every request. Line No 35 to 40. By providing oldRefreshToken.